test/test_tensorboard.py

```diff
@@ -27,6 +27,7 @@
 skipIfNoMatplotlib = unittest.skipIf(not TEST_MATPLOTLIB, "no matplotlib")
 
 import torch
+import torchvision
```

Comment:
Same as `test_jit.py` this needs to still run if someone doesn't have torchvision installed. Follow the pattern there, but it would be something like:

```
try:
    import torchvision
    HAS_TORCHVISION = True
except ImportError:
    HAS_TORCHVISION = False


skipIfNoTorchVision = unittest.skipIf(not HAS_TORCHVISION, "no torchvision")
```